storage/redis/peer_store.go

```diff
@@ -0,0 +1,200 @@
+package redis
+
+import (
+	"encoding/binary"
+	"log"
+	"net"
+	"time"
+
+	"github.com/chihaya/chihaya/bittorrent"
+	"github.com/chihaya/chihaya/storage"
+	redigo "github.com/garyburd/redigo/redis"
+)
+
+const (
+	ipv4 = "4"
+	ipv6 = "6"
+)
+
+// Config holds the configuration of a redis peerstore.
+// KeyPrefix specifies the prefix that could optionally precede keys
+// Instance specifies the redis database number to connect to(default 0)
+// max_numwant is the maximum number of peers to return to announce
```

Comment:
You use the Go field names for `KeyPrefix` and `Instance`, but the YAML key for `max_numwant` :)